src/strategies/math/README.md

```diff
@@ -6,21 +6,68 @@ Currently supported operations are:
 
 - `square-root`
 - `cube-root`
+- `min`
```

Comment:
Can you add documentation on how these min-max work? for square root and cube root it is pretty straightforward

Reply:
Documentation improved by turning the list into a table.